src/less/snackbar-dialog/snackbar-dialog.less

```diff
@@ -3,17 +3,19 @@
 @import "../mixins/private/token-mixins.less";
 
 .snackbar-dialog {
-    .background-color-token(snackbar-dialog-background-color, color-background-secondary);
-    bottom: 0;
+    .background-color-token(snackbar-dialog-background-color, color-background-inverse);
+    .border-radius-token(snackbar-dialog-border-radius, border-radius-100);
+    bottom: 40px;
     box-shadow: 0 0 3px rgba(0, 0, 0, 0.28);
-    .color-token(snackbar-dialog-foreground-color, color-foreground-primary);
-    left: 0;
+    .color-token(snackbar-dialog-foreground-color, color-foreground-on-inverse);
+    left: 8px;
```

Comment:
If I'm not mistaken, --page-grid-gutter-width-small was established to provide a common spacing value although the naming might be too specific? If so, we should probably use the var instead of the raw pixel values for left and right. @agliga , @ianmcburnie , any background on this?

Reply:
Hmm, `--page-grid-gutter-width-small` is specific to page grid module. It is an alias to `@spacing-100`. So we should probably use `@spacing-100` here too (we don't have spacing CSS vars yet).